localization/english/dvg_flavor_genoa_l_english.yml

```diff
@@ -0,0 +1,43 @@
+﻿l_english:
+#Journal Entries
+ #je_caladeran_war_of_independence_spain:0 "The Andino Revolt"
+ #je_caladeran_war_of_independence_spain_reason:0 "The Andino Revolt must be quelled!"
+ 
+ #Events
+ #dvg_spain.1.t:0 "The Andino Revolt!"
+ #dvg_spain.1.d:0 "Treachery!"
+ #dvg_spain.1.f:0 "The Independence of Gran Colombia has sent ripples throughout Arcadia and Ameriga, and these ripples have finally reached New Toledo. They have #abandoned the Spanish Crown and seek independence."
+ #dvg_spain.1.a:0 "¡Viva España!"
+
+ #Modifiers
+ #dvg_giant_big_spain:0 "Gran Imperio Español"
+ 
+ #Names
+ ig_bolognese_school:0 "The Bolognese School"
+ ig_bolognese_school_desc:0 ""
+ ig_genoese_patricians: 0 "Genoese Patricians"
+ ig_genoese_patricians_desc:0 ""
+ Lodovico: "Lodovico"
```

Comment:
Getting error that some of these names are already defined. Make sure to only define them once, somewhere in a localization file. Remove duplicate ones. Also for this reason it's best to keep all names in the dvg_names_ file instead of in flavor localisation.